plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPreferencesPanel.java

```diff
@@ -48,7 +48,7 @@ protected FormToolkit getFormToolkit() {
     return formToolkit;
   }
 
-  private void initializeFormToolkit() {
+  protected void initializeFormToolkit() {
```

Comment:
Small - initializeFormToolkit() does't seem to be used outside this class. Its called in the DeployPreferencesPanel constructor, the GAE panel inherit.